integration/test/metric_value_benchmark/cpu_test.go

```diff
@@ -13,22 +13,16 @@ import (
 	"github.com/aws/amazon-cloudwatch-agent/integration/test/status"
 )
 
-const cpuTestName = "CPU"
-
 type CPUTestRunner struct {
 }
 
-var metricsToFetch = []string{
-	"cpu_time_active", "cpu_time_guest", "cpu_time_guest_nice", "cpu_time_idle", "cpu_time_iowait", "cpu_time_irq",
-	"cpu_time_nice", "cpu_time_softirq", "cpu_time_steal", "cpu_time_system", "cpu_time_user",
-	"cpu_usage_active", "cpu_usage_guest", "cpu_usage_guest_nice", "cpu_usage_idle", "cpu_usage_iowait",
-	"cpu_usage_irq", "cpu_usage_nice", "cpu_usage_softirq", "cpu_usage_steal", "cpu_usage_system", "cpu_usage_user"}
+var _ ITestRunner = (*CPUTestRunner)(nil)
```

Comment:
I did not know this was possible. That is so much better than what I thought I had to deal with, with the implicit interface thing

Reply:
yeah here's a secret: I broke at runtime while writing the cpu test lol. some dumb typo in function names and stuff